### PR TITLE
Pin libcalico-go to version with multiple nets in rule support.

### PR DESCRIPTION
**Pull request by fasaxc**
## Description
Add support for multiple CIDRs in each rule by revving libcalico-go to pick up that change.

## Todos
- [x] Tests
- [x] Documentation
- [x] Release note - handled in other repo

### CODE REVIEW

**Comment by robbrockbank** [comment]
@fasaxc STs need fixing.  We have similar STs in the calicoctl repo that test calicoctl round-tripping data.  Should we be removing these STs and just updating the calicoctl ones?

**Comment by fasaxc** [comment]
I've updated the existing STs to use nets but didn't check the "auto-convert on read" behaviour since that's tested in the calico repo.

Interestingly, these STs did catch a bug in libcalico-go (incorrect JSON name for a field) so hard to argue for their removal.

**Comment by robbrockbank** [comment]
@fasaxc:  Ha!  I think I must have been looking at a couple of different PRs at the same time -I could have sworn this was a calico PR (hence my comment about duplicate tests in calicoctl).

TLDR:  Ignore my comment about removing them!

LGTM.
